client/ayon_harmony/api/lib.py

```diff
@@ -636,3 +638,6 @@ def find_node_by_name(name, node_type):
             return node
 
     return None
+
+def get_workdir():
+    return ProcessContext.workfile_path
```

Comment:
Isnt' the workfile path always kept "live" in AYON regardless, by using `AYON_WORKDIR` since it updates on task change?

So couldn't we technically just rely on `AYON_WORKDIR`?